### DISCUSSION TITLE
Help Needed: Disabling ID Token Encryption for a Client

### DISCUSSION THREAD

**Discussion by bingyulee** [fief-dev/fief#307] [Q&A] [answered]
Hello Fief Community,

I've encountered an issue with ID token encryption in Fief and need some assistance. Recently, I enabled the ID token encryption feature for a specific client in the Fief admin panel. However, now I am unable to disable this feature for the same client.

I have tried looking through the documentation and revisiting the settings in the admin panel, but haven't had any luck. The option to disable the encryption doesn't seem to be available, or I might be missing something. Has anyone else faced this issue? If so, how did you manage to resolve it?

Any guidance or suggestions would be greatly appreciated. I'm looking to understand if this is a common issue and if there are any specific steps I need to follow to disable the ID token encryption.

Thank you in advance for your help!

Best regards,
Bingyu Lee

**Comment by frankie567** [comment, marked as answer]
Hi @bingyulee 👋 

Unfortunately, there is no way to disable ID Token Encryption on a Client once it's enabled. The most straightforward solution is to re-create a new Client.

Another tricky solution, if you have directly access to the SQL database is to set the `encrypt_jwk` column to `NULL` in the `fief_clients` table.

**Reply by bingyulee** [reply]
Hi @frankie567 ,

Thank you so much for your response and the valuable solutions you've provided. I appreciate your insight into the issue with ID Token Encryption, and I find your suggestions very helpful. I will consider both options, particularly the more technical approach involving the SQL database. Your expertise and willingness to help are greatly appreciated!

Best regards,
